src/components/RichText.test.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+
+import RichText from './RichText';
+
+/* Parsing Tests */
+it('Creates <p> elements, given markdown with individual paragraphs', () => {
+  const paragraphMd = 'Hello\n\nWorld!';
+  const expectedHtml = '<p>Hello</p>\n<p>World!</p>';
+  act(() => {
```

Comment:
I am unfamiliar with `act()`. If you could explain to me why it is included in these tests that would be appreciated.

Reply:
Did a little reading and act() is similar to an anonymous async function--assertions are not run until everything in a test's act() call have resolved, like components rendering, user interactions, and so on. It _also_ turns out it's not necessary in our project because React Testing Library's render() function is wrapped in an act() call already 😅